requirements/test.txt

```diff
@@ -4,3 +4,4 @@ pytest
 pytest-django
 pytest-cov
 coverage
+tox
```

Comment:
Yeah, let's put it in dev.txt since the tox examples are many in our developer docs...